### DISCUSSION TITLE
Display Power consumption

### DISCUSSION THREAD

**Discussion by Gyr0sK0mplett** [basti242/homeassistant_lg_therma_v_modbus#12] [General]
Hallo und danke für dieses Projekt.
Ich will unsere Therma V 9kW R32 wahrscheinlich am WE in HA einbinden.
Die Frage ist, kann man auch den Verbrauch bzw. Leistungsaufnahme irgendwie abgreifen? Per Stick von LG ist das ja grundsätzlich möglich.
Ansonsten hoffe ich, dass alles klappt und ich hier nicht mit zig noob-fragen nerven muss 🙈

**Comment by basti242** [comment]
Hallo,

ich habe einen Shelly 3EM verbaut. Dass die Pumpe die Infos rausgeben kann ist mir nicht bekannt. Den Stick von LG habe ich nicht im Einsatz.
Bei Fragen frag einfach, dafür ist das ja alles da.

Gruß
Basti

**Reply by Gyr0sK0mplett** [reply]
Shelly spinnt bei mir leider Rum, zwar nur einphasig, aber spuckt deutlich zu niedrige Werte aus. Elektriker sagt alles richtig.
Egal, gehört hier nicht hin.
Danke, dass du weiterhin ansprechbar bist, da werden sicher Fragen kommen.  Jetzt ist erstmal alles angeschlossen und Dip-Schalter umgelegt, der große Rest folgt häppchenweise.

**Reply by Gyr0sK0mplett** [reply]
Nach einigen Stunden Try and Error kein Erfolg Entitäten nicht verfügbar.
Hardware: Waveshare WiFi, der in der Doku verlinkt ist.
HA Green, hängt an Connect Box.
Ping vom HA auf Waveshare funktioniert.
Ansonsten alles nach Anleitung.
![Screenshot_20241117-011344](https://github.com/user-attachments/assets/e2af2e7d-35cf-4c20-9812-3fae121527f1)
![Screenshot_20241117-012950](https://github.com/user-attachments/assets/85788635-02ba-4d2f-8ec3-9f3aee193adf)
![Screenshot_20241117-011448~2](https://github.com/user-attachments/assets/3fd12a0d-80b1-4a7d-a8dd-34a4506662c7)

**Comment by gRiMMi83** [comment]
Ich greif die Daten über den wlan Stick ab, werd aber auch noch nen Shelly oder was anderes einbauen. Per Modbus hab ich leider nix gefunden, wobei ich das komisch find, de wlan Stick greift die ja such irgendwo ab.

**Comment by basti242** [comment]
Es gibt sicher noch einige undokumentierte Register die nur gefunden werden wollen ;-)

**Comment by Gyr0sK0mplett** [comment]
Lieber Basti, ich hab es schon im Forum geschrieben, nach Verkabelung, also ohne WLAN, funktioniert es. Ich bekomme sämtliche Werte angezeigt. Die Beispielkarten werden zwar einige Fehler aus, aber grundsätzlich funktionieren die wichtigsten Einstellungen. Da wühle ich mich noch in Ruhe durch.
Vielen Dank für das Projekt und vor allem, dass du auch für Fragen zur Verfügung stehst. Ist nicht selbstverständlich.
